test/python/unittest/API/APITest.py

```diff
@@ -308,6 +308,122 @@ def testAttributes(self):
             if openPMD.variants[b] is True and b in backend_filesupport:
                 self.attributeRoundTrip(backend_filesupport[b])
 
+    def makeConstantRoundTrip(self, file_ending):
+        # write
+        series = openPMD.Series(
+            "unittest_py_constant_API." + file_ending,
+            openPMD.Access_Type.create
+        )
+
+        ms = series.iterations[0].meshes
```

Comment:
That object needs love via [return policies](https://pybind11.readthedocs.io/en/stable/advanced/functions.html). Using that shortcut variable corrupts things.

Reply:
Fixed: had to ensure returned `Container`s are freed before the object that exposed them in the garbage collector